ccd-definition/CaseField/CaseFieldDJ-SDO-HNL-nonprod.json

```diff
@@ -368,7 +368,7 @@
   {
     "CaseTypeID": "CIVIL",
     "ID": "trialHearingNotesDJ",
-    "Label": "Notes",
+    "Label": "Important",
```

Comment:
@drummondjm should this not read as "Important Notes" ?

Reply:
Kinda odd phrasing I agree, but it's what was requested

**Notes Section

Change 1 - Amend the title of this section to state "Important" instead of "Notes"**